mmv1/products/vertexai/FeatureGroupFeature.yaml

```diff
@@ -0,0 +1,109 @@
+  guides:
+    'Creating a Feature': 'https://cloud.google.com/vertex-ai/docs/featurestore/latest/create-feature'
+  api: 'https://cloud.google.com/vertex-ai/docs/reference/rest/v1beta1/projects.locations.featureGroups.features'
+async: !ruby/object:Api::OpAsync
+  actions:
+    - create
+    - delete
+    - update
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'response'
+    resource_inside_response: true
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'done'
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error'
+    message: 'message'
+  include_project: true
+description: Vertex AI Feature Group.
+autogen_async: false
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: vertex_ai_feature_group_feature
+    primary_resource_id: feature_group_feature
+    vars:
+      job_id: job_load
+      name: example_feature
+      feature_group_name: example_feature_group
+  - !ruby/object:Provider::Terraform::Examples
+    name: vertex_ai_feature_group_feature_with_version_column_name
+    primary_resource_id: feature_group_feature
+    vars:
+      job_id: job_load
+      name: example_feature
+      feature_group_name: example_feature_group
+    skip_docs: true
+custom_code: !ruby/object:Provider::Terraform::CustomCode
+  pre_create: templates/terraform/constants/vertex_ai_feature_group_feature.go.erb
+  pre_delete: templates/terraform/constants/vertex_ai_feature_group_feature.go.erb
+  custom_import: templates/terraform/custom_import/vertex_ai_feature_group_feature.go.erb
+  extra_schema_entry: templates/terraform/extra_schema_entry/vertex_ai_feature_group_feature.go.erb
+  encoder: templates/terraform/encoders/vertex_ai_feature_group_feature.go.erb
+parameters:
+  - !ruby/object:Api::Type::String
+    name: featureGroup
+    description: The name of the Feature Group to use, in the format
+      `projects/{project}/locations/{location}/featureGroups/{feature_group}`.
+    url_param_only: true
+    immutable: true
+    required: true
+properties:
+  - !ruby/object:Api::Type::String
+    name: name
+    description: The resource name of the Feature Group Feature.
+    required: true
+    pattern: '{{feature_group}}/features/{{name}}'
+    custom_flatten: templates/terraform/custom_flatten/name_from_self_link.erb
+  - !ruby/object:Api::Type::String
+    name: createTime
+    output: true
+    description: The timestamp of when the FeatureGroup was created in RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine fractional digits.
+  - !ruby/object:Api::Type::String
+    name: updateTime
+    output: true
+    description: The timestamp of when the FeatureGroup was last updated in RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine fractional digits.
+  - !ruby/object:Api::Type::String
+    name: etag
```

Comment:
We can omit this or use the ::Fingerprint type for it. Either is fine, Terraform will always want to overwrite the full resource.